delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/operations/s3/client/S3StorageClientDisabled.scala

```diff
@@ -0,0 +1,45 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.operations.s3.client
+
+import akka.http.scaladsl.model.Uri
+import cats.effect.IO
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.model.DigestAlgorithm
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.operations.s3.client.S3StorageClient.{HeadObject, UploadMetadata}
+import ch.epfl.bluebrain.nexus.delta.sdk.error.ServiceError.FeatureDisabled
+import fs2.Stream
+import software.amazon.awssdk.services.s3.model._
+
+private[client] object S3StorageClientDisabled extends S3StorageClient {
```

Comment:
Not to address in this PR and not a priority but we may want to get rid of this one.
As long as you can't create a S3 storage when the feature is disabled, it might be enough ?

Reply:
I agree that this probably shouldn't be needed